src/EditorFeatures/Test2/GoToDefinition/CSharpGoToDefinitionTests.vb

```diff
@@ -111,11 +111,11 @@ class Program
     </Project>
 </Workspace>
 
-            Test(workspace)
-        End Sub
+            Await Test(workspace)
+        End Function
```

Comment:
i hand made all these changes to avoid accidentically changing the contents of any tests :(